app/controllers/api/functions.php

```diff
@@ -1568,3 +1362,165 @@
 
         $response->noContent();
     });
+
+App::get('/v1/functions/:functionId/usage')
```

Comment:
Can you please keep this endpoint back on Line 211 ? Else this diff is hard to interpret .

Reply:
ok